### PR TITLE
[souschef] Use numeric_limits::lowest for min cap

### PR DESCRIPTION
**Pull request by jinevening**
This uses numeric_limits::lowest for min cap.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

**Comment by jinevening** [comment]
std::numeric_limits<float>::min() returns the smallest positive value that float can express. So, negative values are clipped, which is not intended by random generator.

**Comment by seanshpark** [comment]
@binarman PTAL

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Comment by hseok-oh** [comment]
@nnfw-bot test nncc-debug nncc-release

**Comment by seanshpark** [comment]
I'll wait for a day for review from @binarman and if no response, I'll land.

**Review by binarman — APPROVED** [review]
LGTM, thank you!

**Pull request merged**
